pkg/storage/gc_queue.go

```diff
@@ -125,25 +123,40 @@ func (gcq *gcQueue) shouldQueue(
 	zone, err := sysCfg.GetZoneConfigForKey(desc.StartKey)
 	if err != nil {
 		log.Errorf(ctx, "could not find zone config for range %s: %s", repl, err)
-		return
+		return false, 0
 	}
 
+	ttlSeconds := int64(zone.GC.TTLSeconds)
+
+	repl.mu.Lock()
+	considerGC := now.Less(repl.mu.state.GCThreshold.Add((ttlSeconds*1E9)/2, 0))
+	repl.mu.Unlock()
+
 	ms := repl.GetMVCCStats()
-	// GC score is the total GC'able bytes age normalized by 1 MB * the replica's TTL in seconds.
-	gcScore := float64(ms.GCByteAge(now.WallTime)) / float64(zone.GC.TTLSeconds) / float64(gcByteCountNormalization)
+	// GC score is the total GC'able bytes age normalized by (replica size * the replica's TTL in seconds).
+	// TODO(tschottdorf): ttlSeconds = 0 || ValBytes == 0?
+	denominator := float64(ttlSeconds * ms.ValBytes)
```

Comment:
actually this should be `GCBytes()`. You're probably best off just looking at the PR description for now. I'm also going to have to do something about the case in which there isn't much to GC, to make sure we don't even bother to GC in that case.